package.json

```diff
@@ -29,6 +29,9 @@
     "babel-preset-es2015": "^6.24.1",
```

Comment:
You may want to remove duplicate dependencies in Dependencies and devDependencies